### ISSUE TITLE
issue building indicators

### ISSUE THREAD

**Issue by adcdam** [hcsubser/hybridbar#10]
im trying to build the package, the package without indicators build fine but when i try to build indicator it cant find "hybridbar-1.0" because its not already installed, i dont have the problem if i build and install hybridbar first without indicators. But this is not correct it should all build at the same time or have another package for indicators if i want to package this for the distro i use.

mkdir build
cd build
  CFLAGS="$SLKCFLAGS" \
  CXXFLAGS="$SLKCFLAGS" \
  meson .. \
    --buildtype=release \
    --infodir=/usr/info \
    --libdir=/usr/lib${LIBDIRSUFFIX} \
    --localstatedir=/var \
    --mandir=/usr/man \
    --prefix=/usr \
    --sysconfdir=/etc \
    -Dstrip=true \

   "${NINJA:=ninja}"
  DESTDIR=$PKG $NINJA install

cd ..
cd indicators
./build.sh --prefix=/usr

**Comment by hcsubser** [comment]
What distribution is this on? And what prefix did you set when compiling hybridbar?
It is saying that it cannot find hybridbar-1.0.pc file which is generally located under $PREFIX/$LIB/pkg-config.

If you just ran 'meson build' to compile then prefix is /usr/local and on Ubuntu or Debian that pkg-config file will be under /usr/local/lib/x86_64-linux-gnu/pkg-config on arch and I believe Fedora it will be under /usr/local/lib64/pkg-config.

If your prefix is for example /opt/hybridbar then it will be located here:
Ubuntu,Debian:/opt/hybridbar/lib/x86_64-linux-gnu
Fedora,Arch:/opt/hybridbar/lib64/pkg-config
you get the idea.

what you have to do, because meson and pkgconfig do not seem to be able to find that hybridbar-1.0.pc file you have to find the folder that it is located in yourself and run ./build.sh like this:
PKG_CONFIG_PATH=/path/to/lib/pkg-config ./build.sh
NOTE:Path to folder that .pc file is located in, not the file itself

**Comment by hcsubser** [comment]
Alternatively you could compile hybridbar with `meson build --prefix=/usr` and then do the same with indicators

**Comment by hcsubser** [comment]
Yes I recognize that this is not correct but that is how currently is done.
First you compile hybridbar and then indicators. I should probably make these seperate packages but I really have not had enough free time to work on these hobby projects lately. I will try to split the panel and indicators into seperate repos so that you and anybody else can make distro packages. I will let you know here when I'm done.

**Comment by adcdam** [comment]
Thank you very much!!! having the indicators in separate packages would be great!!
best regards!!

**Comment by adcdam** [comment]
Please dont forget to split  the indicators, the program is very nice. i really want to package this for Slackware.
